### PR TITLE
feat: add styling to the about page

### PR DESCRIPTION
**Pull request by jdiogopeixoto**
- Adds styling to the about page
- Changes tailwind typography to use the global typography for the markdown
- Updates the footer links according to the Figma design
- Makes the footer responsive in narrower screens
- Adds a global `layout-margins` class
- Updates the login page to use the layout class

### ✨ Screenshot

image

### CODE REVIEW

File: packages/website/pages/about.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 /* eslint-disable react/no-children-prop */
 import matter from 'gray-matter'
 import ReactMarkdown from "react-markdown";
+// @ts-ignore
```
Comment (rafaelramalho19) [inline packages/website/pages/about.js:4]: An alternative for this is to make the vertical-lines a javascript file and also let it receive props

Comment (rafaelramalho19) [reply packages/website/pages/about.js:4]: See the icons/github.js for more info

**Comment by rafaelramalho19** [comment]
It's really hard to read in mobile
image


Maybe just remove the illustration for mobile?

File: packages/website/pages/about.js (hunk 1 above)
Comment (jdiogopeixoto) [reply packages/website/pages/about.js]: Done ✅

**Comment by jdiogopeixoto** [comment]
@rafaelramalho19 I fixed the illustration, it should be good now.

**Review by alanshaw — APPROVED** [review]

**Review by rafaelramalho19 — APPROVED** [review]

**Pull request merged**